lib/config.cpp

```diff
@@ -533,7 +536,7 @@ schemas Config::load_schemas(std::string schemas_dir) {
     BOOST_LOG_FUNCTION();
     schemas schemas;
 
-    EVLOG(info) << "Loading base schema files for config and manifests... from:" << schemas_dir;
+    EVLOG(info) << fmt::format("Loading base schema files for config and manifests... from:", schemas_dir);
```

Comment:
Not sure if this might work or not, but it surely would be more clear to put the replacement operators `{}` in at the end of the string.
Please check!

Reply:
:+1: for your findings! Thanks!!

Reply:
You're welcome!

Reply:
Resolved!